File: Core/Shared/JavaLib/src/net/lukx/jchatter/lib/comms/LoginHeader.java
package net.lukx.jchatter.lib.comms;

import com.sun.istack.internal.NotNull;
import net.lukx.jchatter.lib.PublicApi;

/***
 * Header needed to use some of Chatter's
 * endpoints
 */
@PublicApi
public class LoginHeader {

    private String Login;
    private String Password;

    /**
     * @return login
     */
    public String getLogin() {
        return Login;
    }

    /**
     * @param login sets
     */
    @PublicApi
    public void setLogin(@NotNull String login) {
        Login = login;
    }

    /**
     * @return password
     */
    @PublicApi
    public String getPassword() {
        return Password;
    }

    /**
     * @param s password
     */
    @PublicApi
    public void setPassword(@NotNull String s) {
        Password = s;
    }
}

File: Core/Wpf/ChatterWpf/ChatterWpf/Com/Comms/LoginHeader.cs
using System;
using System.String;

//HELP
public class LoginHeader
{

    private string Login;
    private string Password;

    public string getLogin()
    {
        return Login;
    }

    [PublicApi]
    public void setLogin(/*String.IsNullOrEmpty*/string login)
    {
        Login = login;
    }

    [PublicApi]
    public string getPassword()
    {
        return Password;
    }

    [PublicApi]
    public void setPassword(/*@NotNull */string password)
    {
        Password = password;
    }
}
